build.fsx

```diff
@@ -305,10 +297,12 @@ Target "CI" DoNothing
 
 "RunTests"
   =?> ("SourceLink", isLocalBuild && not isLinux)
-  ==> "NuGet"
+  ==> "PaketPack" 
+  ==> "PaketPush"
   ==> "Release"
 
-"GenerateDocs" ==> "CI"
-"NuGet" ==> "CI"
+"GenerateDocs"
+  ==> "PaketPack" 
+  ==> "CI"
```

Comment:
Not a big deal but I like to have separate "generate nuget packages" and "generate documentation" targets, i.e. if I'm checking the packaging I don't necessarily want to generate the docs.